core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -1144,4 +1148,28 @@ class TestSuiteWriter {
             .toList()
     }
 
+    private fun assertionUtilFunctionForSSRF(lines: Lines, format: OutputFormat) {
+        lines.startCommentBlock()
+        lines.addBlockCommentLine("A method for verifying requests sent to HttpCallbackVerifier.")
+        lines.endCommentBlock()
+        when {
+            format.isKotlin() -> {
+                lines.add("fun verifierHasServedRequests(verifier: WireMockServer, actionName: String) : Boolean")
```

Comment:
the name `verifierHasServedRequests`  should be in a static constant, as used in another file as well. also, might want to rename it into `hasReceivedAnyRequests`